pkg/utils/fs/fs.go

```diff
@@ -236,12 +236,7 @@ func walkDir(fs v1.FS, path string, d fs.DirEntry, walkDirFn fs.WalkDirFunc) err
 }
 
 func readDir(fs v1.FS, dirname string) ([]fs.DirEntry, error) {
-	f, err := fs.Open(dirname)
-	if err != nil {
-		return nil, err
-	}
-	dirs, err := f.ReadDir(-1)
-	f.Close()
+	dirs, err := fs.ReadDir(dirname)
```

Comment:
now it directly opens the dir, this function is mainly obsolete as it does the same as the underlying!